pkg/operator/ceph/file/mirror/mirror.go

```diff
@@ -93,24 +88,24 @@ func (r *ReconcileFilesystemMirror) start(filesystemMirror *cephv1.CephFilesyste
 
 	if _, err := r.context.Clientset.AppsV1().Deployments(filesystemMirror.Namespace).Create(ctx, d, metav1.CreateOptions{}); err != nil {
 		if !kerrors.IsAlreadyExists(err) {
-			return errors.Wrapf(err, "failed to create %q deployment", resourceName)
+			return errors.Wrapf(err, "failed to create %q deployment", AppName)
```

Comment:
Instead of using `AppName` in all these message, how about `d.Name`?
```suggestion
			return errors.Wrapf(err, "failed to create %q deployment", d.Name)
```